index.html

```diff
@@ -0,0 +1,185 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>BHROMAON</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+  <body>
+    <section class="search">
+      <h1 class="search__heading heading1">BHROMAON</h1>
+      <p class="search__subheading subheading">Safe Travel Anywhere</p>
+      <div class="search__bar">
+        <input
+          type="text"
+          name="search"
+          id=""
+          class="search__search-input"
+          placeholder="Search Your Travel Destination"
+        />
+        <button type="button" class="search__button heading4">
```

Comment:
этой кнопкой ты отправляешь форму, поэтому **submit**
```suggestion
        <button type="submit" class="search__button heading4">
```